scripts/publishToJfrog.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+result=${PWD##*/}
+if [[ "$result" = "scripts" ]]
+then
+    echo "script must be run from root project folder, not $PWD"
+    exit 1
+else
+    echo "we are in $PWD and tag is $RELEASE_TAG"
+
+    if [[ $RELEASE_TAG =~ ^[0-9]+\.[0-9]+\.[0-9]+$ ]]
+    then
+        echo "publishing: tag $RELEASE_TAG looks like a semver"
+        git status
+        git describe --tags
+        ./gradlew printVersion
+        ./gradlew publishMyPublicationPublicationToLinkedInJfrogRepository
```

Comment:
Is this goal implemented already?  I didn't see it in the PR

Reply:
this goal/task is dynamic: gets generated from `build.gradle` publication declarations. 
IE [`LinkedInJfrog`](https://github.com/linkedin/kafka-monitor/pull/374/files#diff-49a96e7eea8a94af862798a45174e6ac43eb4f8b4bd40759b5da63ba31ec3ef7R128) becames part of the goal name